azure-pipelines.yml

```diff
@@ -60,7 +60,7 @@ stages:
         pip install --user --upgrade --progress-bar off "ipython!=8.7.0" numpy seaborn statsmodels matplotlib sphinx pillow pytest pytest-cov joblib "plotly>=4.0"
         pip install --user --upgrade --progress-bar off --pre pydata-sphinx-theme
         echo "Qt, plotly, VTK"
-        pip install --user --upgrade --progress-bar off pyvista "pyqt5!=5.15.8" plotly vtk jupyterlite-sphinx
+        pip install --user --upgrade --progress-bar off pyvista "pyqt5!=5.15.8" plotly vtk jupyterlite-sphinx jupyterlite-pyodide-kernel
```

Comment:
Done. We don't pin anything else in the sphinx-gallery CI so we like to live dangerously it seems but we can make an exception for jupyterlite-sphinx :wink:.

> We are working on getting JupyterLite 0.1.0 out very soon and things. I can open a new issue / PR on this repo to do the update.

Sounds great, don't hesitate to ping me on JupyterLite-related things, otherwise I may well miss it completely!